### PR TITLE
chore: Update spelling baseline

### PR DESCRIPTION
**Pull request by ellismg**
Add some missing words to our custom dictionary to get the linter clean again.

### CODE REVIEW

File: cli/azd/.vscode/cspell-azd-dictionary.txt [hunk 1]
```diff
@@ -2,9 +2,13 @@ AADSTS
 ABRT
 aiomysql
 aiopg
+aistudio
+alexwolf
```
Comment (vhvb1989) [inline cli/azd/.vscode/cspell-azd-dictionary.txt:6]: some of this is already in the config. Like: https://github.com/Azure/azure-dev/blob/main/cli/azd/.vscode/cspell.yaml#L39

Some cases are covered per file only.

**Comment by vhvb1989** [comment]
Did you create this based on https://github.com/Azure/azure-dev/actions/runs/10207492464/job/28242344666?pr=4160 ?   

I think there's something going on with cspell. I'm looking into it

**Comment by vhvb1989** [comment]
Yeah, looks like there's some sort of regression on cspell 8.13.0 (released 3 days ago) https://github.com/streetsidesoftware/cspell/releases/tag/v8.13.0.

Using `8.12.1` works fine with our cspell config, but `8.13.0` ignores our config

**Comment by vhvb1989** [comment]
Created an issue for cspell: https://github.com/streetsidesoftware/cspell/issues/6025

**Comment by vhvb1989** [comment]
See: https://github.com/Azure/azure-dev/pull/4164

**Review by vhvb1989 — CHANGES_REQUESTED** [review]
adding request-changes only b/c I saw auto-merge :P

There is a bug on latest cspell lib, we should pin one version before until they fix it, instead of updating all the dictionaries. See: https://github.com/Azure/azure-dev/pull/4164

**Comment by ellismg** [comment]
Sounds good, I'll change to pin cspell.

**Pull request closed without merging**